Makefile

```diff
@@ -28,7 +24,9 @@ LDFLAGS += -Wl -L/usr/local/opt/openal-soft/lib
 CFLAGS += -stdlib=libc++ -DOSX -mmacosx-version-min=10.7
 CFLAGS += -I/usr/local/opt/openal-soft/include
 else
-LDFLAGS += -Wl,-rpath=$(RPATH)
+ifndef NO_RPATH
+LDFLAGS += -Wl,-rpath=$(or $(RPATH),.)
```

Comment:
I'm not a Makefile expert, but this `$(or A,B)` conditional seems to do the trick to allow defining `RPATH` or defaulting to `.` if not defined.